core/src/test/java/org/everit/json/schema/loader/JsonPointerEvaluatorTest.java

```diff
@@ -94,9 +94,21 @@ public void remoteDocumentSuccess() throws URISyntaxException {
         assertEquals(new SchemaLocation(new URI("http://localhost:1234/hello"), asList("definitions", "Bar")),
                 actual.ls.pointerToCurrentObj);
     }
+    
+    @Test
+    void remoteDocument_jsonParsingFailure() {
+        SchemaClient schemaClient = mock(SchemaClient.class);
+        when(schemaClient.get("http://localhost:1234/hello")).thenReturn(asStream("unparseable"));
+        try {
+            JsonPointerEvaluator.forURL(schemaClient, "http://localhost:1234/hello#/foo", createLoadingState(schemaClient, "")).query();
+            fail("did not throw exception for unparseable json");
+        } catch (SchemaException e) {
```

Comment:
As junit5 has builtin assertThrows method maybe it is better to use that for testing exceptions.

Reply:
Makes sense, fixed.